exercises/J-array-get-set/exercises2.js

```diff
@@ -7,7 +7,8 @@
 */
 
 let numbers = [1, 2, 3]; // Don't change this array literal declaration
-
+numbers[numbers.length] = 4;
+ numbers[3] = 4;
```

Comment:
You're assigning the same thing twice here; either of these will do but you don't need both.

Reply:
You could also use `numbers.push(4)`